apps/myaccount/src/configs/routes.tsx

```diff
@@ -20,28 +20,7 @@ import { HomeIcon, ShieldCheckIcon, TilesIcon, UserDocumentIcon } from "@oxygen-
 import { RouteInterface } from "@wso2is/core/models";
 import React,{ FunctionComponent, lazy } from "react";
 import { AppConstants } from "../constants";
-import { AppLayout, AuthLayout, DashboardLayout, DefaultLayout, ErrorLayout } from "../layouts";
-
-/**
- * Get default page layout routes.
- *
- * @returns Default Layout routes.
- */
-export const getDefaultLayoutRoutes = (): RouteInterface[] => {
-
-    return [
```

Comment:
Lets just have `empty` array here and keep the default layout implementation just in case if we have similar sort of layouting requirements for a page.